workflows/tomte.nf

```diff
@@ -22,6 +22,7 @@ include { CALL_VARIANTS           } from '../subworkflows/local/call_variants'
 include { ALLELE_SPECIFIC_CALLING } from '../subworkflows/local/allele_specific_calling'
 include { ANNOTATE_SNV            } from '../subworkflows/local/annotate_snv'
 include { IGV_TRACKS              } from '../subworkflows/local/igv_tracks'
+//include { BUILD_DROP_DATABASE     } from '../subworkflows/local/drop_database'
```

Comment:
Why is this commented out?

Reply:
I was thinking of making a separate module to build the database but in the end I just went for modifying the existing one, so there is no such module, I will remove that part